openreferee_server/server.py

```diff
@@ -215,7 +217,14 @@ def create_editable(
       responses:
         200:
           description: Editable processed
+          content:
+            application/json:
+              schema: CreateEditableResponseSchema
     """
+    resp = {}
+    if not PROCESS_EDITABLE_FILES:
+        resp['ready_for_review'] = True
```

Comment:
```suggestion
    resp = {'ready_for_review': not PROCESS_EDITABLE_FILES}
    if not PROCESS_EDITABLE_FILES:
```

Maybe being more explicit is better here?